pipeline/.env.example

```
@@ -31,6 +31,6 @@ SPACE_DB_PASS={{SPACE_DB_PASS}}
 SPACE_LOG_LEVEL=warning
 SPACE_PORT=8083
 SPACE_CAN_CREATE=true
-SPACE_DEFAULT=default
+SPACE_DEFAULT=key1
```

Давай в этом файле по умолчанию сделаем этот конфиг пустым.
```suggestion
SPACE_DEFAULT=
```